CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - Update rules model to allow tracking of log_event ARNs related to
     Rule event logging.    Kinesis rule types will now automatically log
     incoming events via a Kinesis event triggered lambda.
+  - Update migration code to require explicit migration names per run
```

Comment:
We need to call out in the **Please Note** section that redeployment of IAMs is required. I'd make that the top item.

Reply:
49d1a58e